README.md

```diff
@@ -257,6 +257,29 @@ const record = await linnia.addRecord(dataHash, metadata, dataUri, ethParams);
 
 
 
+# Attest data on Linnia
+
+Attest data on Linnia means sign a record and verify that contain legitimate and correct information.
+
+## linnia.signRecord
+
+```javascript
+linnia.signRecord(dataHash, ethParams);
+```
+
+Add record to Linnia
+
+### Parameters
+
+1. `String` - The data hash, hex-encoded, 0x prefixed
+2. `Object` - The ethParams ethereum account params. 
```

Comment:
Optional

Reply:
Won't it default to `accounts[0]` if `from` isnt provided? It does serverside at least

Reply:
Let me try if it works